src/main/java/ch/njol/skript/effects/EffDetonate.java

```diff
@@ -0,0 +1,56 @@
+package ch.njol.skript.effects;
+
+import org.bukkit.entity.*;
+import org.bukkit.entity.minecart.ExplosiveMinecart;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
```

Comment:
I believe Skript is using Jetbrains annotations now.

Reply:
ah my bad

Reply:
yes and every other effect needs to be changed